contracts/sweep/BaseTokenSweep.sol

```diff
@@ -0,0 +1,30 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity ^0.8.0;
+
+import "@openzeppelin/contracts-upgradeable/proxy/utils/UUPSUpgradeable.sol";
+
+/**
+ * @title Abstract upgradeable contract that provides the ability to sweep erroneous transferred tokens to a defined beneficiary address
+ *
+ * @dev Access control implementation is required for many functions by design
+ */
+abstract contract BaseTokenSweep is UUPSUpgradeable {
+    address public beneficiary;
```

Comment:
issue: keep the variables private, providing getter & setters as needed.

We're following the latest OZ switch to keeping contract variables privately scoped (rather than public). 
In practice it only really affects other contracts interacting, but in theory it closer adherence to principle of encapsulation.

In practice that means
```
address private _ beneficiary
``` 

```
function beneficiary() external view returns(address) {
    return _ beneficiary;
}
```

There's no difference in deployment cost as public variables have getters & setters auto-generated for them on compilation (EVM bytecode roughly equivalent) ...but you can add awesome comments to your getters 😉